modules/nw-egress-ips-about.adoc

```diff
@@ -80,22 +80,35 @@ image::nw-egress-ips-diagram.svg[Architectural diagram for the egress IP feature
 
 Both Node 1 and Node 3 are labeled with `k8s.ovn.org/egress-assignable: ""` and thus available for the assignment of egress IP addresses.
 
-The following `EgressIP` object describes a configuration that selects all pods in the `namespace1` namespace, with the `192.168.126.10` and `192.168.126.102` egress IP addresses specified.
+The `namespace1` namespace is associated with the following project definition:
+
+.Namespace object with `name=namespace1`
```

Comment:
This is a small nit: but it might be worth specifying two namespaces for the sake of this example. I am saying this because it's the "power" of labels, i.e: providing the user a higher level abstraction to select things on. It might be better to explicitly show an example of that, where multiple namespaces can be selected by the same label.

Reply:
@alexanderConstantinescu, that's a good idea, I'll include that in an update!